addons/binding/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/BaseChannelHandler.java

```diff
@@ -46,12 +46,7 @@ public void onSuccess(R object) {
     };
 
     // IP to Subscriptions map
-    private Map<String, ServiceSubscription<T>> subscriptions = new ConcurrentHashMap<>();
-
-    // lazy init
-    private synchronized Map<String, ServiceSubscription<T>> getSubscriptions() {
-        return subscriptions;
-    }
+    private Map<String, @Nullable ServiceSubscription<T>> subscriptions = new ConcurrentHashMap<>();
```

Comment:
Can you leave out this `@Nullable` we cannot add null values, handling this is a flaw around us missing proper annotations in the JDK :-( So yes you have to check for `null` when calling the get on the map but  no you shall not save it using this annotation here.

Reply:
Done